cartographer/mapping/map_builder.cc

```diff
@@ -327,5 +361,10 @@ int MapBuilder::num_trajectory_builders() const {
 
 PoseGraphInterface* MapBuilder::pose_graph() { return pose_graph_; }
 
+const std::vector<proto::TrajectoryBuilderOptionsWithSensorIds>&
+MapBuilder::GetAllTrajectoryBuilderOptions() const {
```

Comment:
Is this currently unused?

Reply:
If there are no concrete plans to use it in a follow-up PR please remove it.

Reply:
I don't have any concrete plans to use it inside libcartographer. I believe though it should be a part of the interface. E.g. the library user loads a `.pbstream` and wants to inspect/reuse the trajectory builder options used to build the loaded trajectories.

Reply:
@cschuet ping

Reply:
Agreed.